discord/application.go

```diff
@@ -131,3 +131,10 @@ type InstallParams struct {
 	// Permissions is the permissions to request for the bot role.
 	Permissions Permissions `json:"permissions,string"`
 }
+
+type ApplicationIntegrationType uint16
```

Comment:
Chose `uint16` over `uint8` because slices of `uint8` are treated as byte slices